OpenOversight/app/utils.py

```diff
@@ -95,6 +96,18 @@ def allowed_file(filename):
            filename.rsplit('.', 1)[1].lower() in current_app.config['ALLOWED_EXTENSIONS']
 
 
+def requires_conversion(filename):
```

Comment:
It would be dandy (I'm bringing this word back) to have a unit test for `requires_conversion`. It could be modeled off `test_user_can_submit_allowed_file` below - the asserts would verify that the correct result is returned by the function for files that do and do not require conversion.